Library/HUMSlider.m

```diff
@@ -278,6 +504,8 @@ - (void)sliderAdjusted
 {
     CGFloat halfValue = (self.minimumValue + self.maximumValue) / 2.0f;
     
+    //TODO: Do we need to do anything special here? What does this do?
```

Comment:
This just changes which image is saturated at either end of the track. It doesn't really have anything to do with the track itself.